frontend/src/assets/css/color-variables.css

```diff
@@ -0,0 +1,29 @@
+:root {
+    /* Background */
```

Comment:
make just colors. We don't need to specify it for fonts and background

Reply:
Done in commit [03db74d](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/03db74d7ec35888e47792a6337700fc5e0d1a4a5)